docs/howto/connection/spark.rst

```diff
@@ -0,0 +1,43 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Apache Spark Connection
+=======================
+
+The Apache Spark Connection type enables connection to Spark cluster.
```

Comment:
```suggestion
The Apache Spark Connection type enables connection to Apache Spark.
```
I think this operator also work on single node, correct me if I am wrong

Reply:
You are right, thank you @zhongjiajie.